aezeed/bench_test.go

```diff
@@ -0,0 +1,65 @@
+package aezeed
+
+import (
+	"testing"
+	"time"
+)
+
+var (
+	mnenonic Mnenonic
+
+	seed *CipherSeed
+)
+
+// BenchmarkFromMnenonic benchmarks the process of converting a cipher seed
+// (given the salt), to an enciphered mnenonic.
+func BenchmarkToMnenonic(b *testing.B) {
+	scryptN = 32768
+	scryptR = 8
+	scryptP = 1
+
+	pass := []byte("1234567890abcedfgh")
+	cipherSeed, err := New(0, nil, time.Now())
+	if err != nil {
+		b.Fatalf("unable to create seed: %v", err)
+	}
+
```

Comment:
Add `b.ReportAllocs()` here so that we can measure bytes/allocs per invocation?

Reply:
Fixed.

Reply:
Added.